From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: way-zer <himc.wicp@gmail.com>
Date: Fri, 23 Jun 2023 16:20:12 +0800
Subject: [PATCH] S: add `Config.locale` for localizedName

---
 core/src/mindustry/net/Administration.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/src/mindustry/net/Administration.java b/core/src/mindustry/net/Administration.java
index d544531fac41023e5d71ddcbdabb493118dfb35e..1fb21851e5c95530cf549d9a24e3a353c2b1dd7a 100644
--- a/core/src/mindustry/net/Administration.java
+++ b/core/src/mindustry/net/Administration.java
@@ -488,6 +488,7 @@ public class Administration{
         serverName = new Config("name", "The server name as displayed on clients.", "Server", "servername"),
         desc = new Config("desc", "The server description, displayed under the name. Max 100 characters.", "off"),
         port = new Config("port", "The port to host on.", Vars.port),
+        locale = new Config("locale", "The locale for localizedName.", "default", "locale"),
         autoUpdate = new Config("autoUpdate", "Whether to auto-update and exit when a new bleeding-edge update arrives.", false),
         showConnectMessages = new Config("showConnectMessages", "Whether to display connect/disconnect messages.", true),
         enableVotekick = new Config("enableVotekick", "Whether votekick is enabled.", true),
